components/murata/NCP03WF104F05RL.stanza

```diff
@@ -27,16 +27,15 @@ public pcb-component component :
   val pkg = ipc-two-pin-landpattern("0201")
   landpattern = pkg(p[1] => pkg.p[1], p[2] => pkg.p[2])
 
-  property(p[1].rated-voltage) = 5.0
-  property(p[2].rated-voltage) = 5.0
+  property(self.rated-voltage) = 5.0
```

Comment:
This pattern assumes that the component has the property and not the pins - which may also be a necessary use case when the rated-voltage on pins could be different. In this case, I assume that the component has the relevant constraint.

Reply:
That's right - this matches the model of parts from the database and is valid at least for two-pin non polarized components

Reply:
Great. Let's make sure that our checking logic looks for both component ratings as well as pin ratings then.